modules/travel_pay/spec/services/service_error_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TravelPay::ServiceError do
+  context 'raise_mapped_error' do
+    let(:error) do
+      env = Faraday::Env.new.tap do |e|
+        e.status = 500
+        e.define_singleton_method(:response_body) { nil }
+      end
+      Faraday::Error.new('There was a problem', env)
+    end
+
```

Comment:
Can we also test the `else` case, where it simply raises a `Common::Exceptions::ServiceError`?

Reply:
Based on the feedback above, I've removed the `else` condition, but also added a test for the normal use case.